src/less/breadcrumbs/ds4/breadcrumbs.less

```diff
@@ -7,4 +7,11 @@
 @breadcrumbs-item-focus-hover-color: @color-core-blue;
 @breadcrumbs-item-current-color: @color-core-gray-dim;
 
+// CSS Custom Properties for DS6 only
```

Comment:
You can remove the properties from this DS4 file. We are doing custom properties and dark mode for DS6 only.

Reply:
Actually, I see they are missing from the ds6 file, so just move these over to ds6.

Reply:
Doh! I put it in the wrong place! Will put it in the right place.

Reply:
And fixed!